tests/update/updates_packagekit_kde.pm

```diff
@@ -51,7 +51,13 @@ sub run {
             # the assert_screen, record a soft failure
             wait_still_screen;
             if (match_has_tag('updates_none')) {
-                if (check_screen 'updates_none', 30) {
+                #Check if the applet reports that a restart is needed.
+                if (match_has_tag('updates_installed-restart')) {
```

Comment:
Having both `updates_none` and `updates_installed-restart` tags in the same needle seems odd

Reply:
I seperated.